runtime/doc/eval.txt

```diff
@@ -1195,6 +1195,7 @@ When expr8 is a |Funcref| type variable, invoke the function it refers to.
 expr8->name([args])	method call			*method* *->*
 expr8->{lambda}([args])
 
+							*E260*
```

Comment:
could also add E276: https://github.com/vim/vim/commit/2f0936cb9a2eb026acac03e6a8fd0b2a5d97508b#diff-b3aeef96275167ddcc1e5fee6d51b8f2018f9359f921d86ad6e7035ca80f20e9R1349, though it's not from this rt update (it's from an rt update from before method call syntax was ported so I had missed it)